dmcp/bcd.py

```diff
@@ -212,9 +212,9 @@ def proximal_op(prob, var_slack, lambd):
         # add quadratic terms for all variables that are not slacks
         if not var.id in slack_id:
             if prob.objective.NAME == 'minimize':
-                new_cost = new_cost + cvx.square(cvx.norm(var - var.value,'fro'))/2/lambd
```

Comment:
fro should still work.

Reply:
Should I add a case for both 'fro' and 2? If I use 'fro' for a non-matrix variable, I get an error since 'fro' is not a valid input for when x.ndim < 2.